ts/IterableInboxMessageListItem.tsx

```diff
@@ -139,15 +141,14 @@ const defaultMessageListLayout = (
 
 const IterableInboxMessageListItem = ({ 
    last,
+   dataModel,
    rowViewModel,
    customizations,
    messageListItemLayout,
    isPortrait
 }: MessageListItemProps) => {
    return(
-      messageListItemLayout(last, rowViewModel) ?
-         messageListItemLayout(last, rowViewModel) :
-         defaultMessageListLayout(last, rowViewModel, customizations, isPortrait)  
+      messageListItemLayout(last, rowViewModel) ? messageListItemLayout(last, rowViewModel) : defaultMessageListLayout(last, dataModel, rowViewModel, customizations, isPortrait)  
    )
```

Comment:
It seems like the data model is only used in the IterableInboxMessageListItem component for the formatted time stamp. Do we need to be importing the data model in the message list, swipeable row, and clickable row? To me it seems like it makes sense to only import it where it is used unless we anticipate it will be used in the other components in the future.

Reply:
You're right, but right now it's all structured in a nested manner. Because of this, I ended up creating a new ticket to create a new `IterableInboxCell` component that can intake all of those (and the data model), and also has the swiping,  clicking and item layout inside it, but that's a much bigger effort.